client/lib/posts/post-list-store.js

```diff
@@ -103,14 +103,16 @@ export default function( id ) {
 	}
 
 	/**
-	 * Sort the active list
-	 **/
-	function sort() {
+	 * sort post-list according to current orderBy parameter
+	 * @param  {array} list - list of posts to sort
+	 * @return {array}      sorted array (original will not be mutated)
```

Comment:
Can we be a little more flexible about jsdoc documentation? I'm not sure everyone has converged on http://usejsdoc.org as a standard, and I'd rather retain the use of the tooling (using docblockr) than have to ditch the tool and write the blocks manually to match the specs. I see within calypso other people seem to be using different tooling as well, as I see a number of different flavors of jsdoc, but they all seem understandable.